Cargo.lock

```diff
@@ -1,6 +1,6 @@
 [root]
 name = "cargo-edit"
-version = "0.1.6"
+version = "0.2.0"
```

Comment:
There's https://github.com/jonas-schievink/cargo-tidy, but that doesn't look very featured at a glance.